weave/trace_server/clickhouse_trace_server_batched.py

```diff
@@ -244,6 +246,13 @@ def file_storage_client(self) -> Optional[FileStorageClient]:
         self._file_storage_client = maybe_get_storage_client_from_env()
         return self._file_storage_client
 
+    @property
+    def kafka_producer(self) -> KafkaProducer:
+        if self._kafka_producer is not None:
+            return self._kafka_producer
+        self._kafka_producer = KafkaProducer.from_env()
```

Comment:
Is this going to have problems for environments that don't have kafka setup? should this be a conditional return?

Reply:
This will only be called if `WF_ENABLE_ONLINE_EVAL` is `true`.